server/controllers/events/events_controller.go

```diff
@@ -626,6 +626,12 @@ func (e *VCSEventsController) HandleAzureDevopsPullRequestCommentedEvent(w http.
 		e.respond(w, logging.Debug, http.StatusOK, "Ignoring comment event since no comment is linked to payload; %s", azuredevopsReqID)
 		return
 	}
+
+	if *resource.Comment.IsDeleted {
```

Comment:
I would think you'd need to add this `if` block in the AzureDevops VCS section. It's odd that this function exists in the shared events_controller but this if block makes sense where it is then. 

I second @jamengual's suggestion to add a test.

We do not currently have any azure devops tests regarding comments but we do for bitbucket and github. If you can follow those as an example, we'd really appreciate azure devops tests.

https://github.com/runatlantis/atlantis/blob/585ce1e888f94c83a8774e56a9ef310c1d6066d7/server/events/event_parser_test.go#L1020

Could you add this for azure devops @SSKLCP to validate your change so it doesn't break in the future?